.github/workflows/autoprerelease.yml

```
@@ -19,7 +19,8 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
         with:
-          ref: 'main'
+        ref: ${{ github.event.pull_request.head.ref }}
```

actionlint (1.7.7)

unexpected key "ref" for "step" section. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "uses", "with", "working-directory"

(syntax-check)